mscore/uploadscoredialog.cpp

```diff
@@ -136,6 +136,8 @@ void UploadScoreDialog::upload(int nid)
            QString privateString = cbPrivate->isChecked() ? "1" : "0";
             _loginManager->upload(path, nid, title->text(), description->toPlainText(), privateString, licenseString, tags->text(), changes->toPlainText());
            }
+     // delete the temp file created above
+     QFile::remove(path);
```

Comment:
Darn, doesn't work at all, the file seems to be still open, can't even delete it from the file manager (Windows Explorere in this case), Can delete the file only after MuseScore stopped. So there seems a real bug lurking, files do get opened but not closed explicitly, only implicitly when MuseScore stops?

Reply:
Not addressed review, right? Does it lead to inconsistencies now?

Reply:
Well, it is my own review ;-)
IIRC (it's been quite long ago) that `remove()` didn't work, i.e. it didn't remobe that file, but doesn't cause any other issues (its'return value, here 'failure' is silently ignored).
It should work on Mac and Linux (which can remove open files) though. 
Not sure what to do on Windows, whether there's some 'remove on close'?

Reply:
Maybe here too `setAutoRemove(true)` would help?

Reply:
Hmm, no, that only existe for `QTemporaryFile`, not for `QFile` it seems.

Reply:
The file in the ```_loginManager->upload()``` is uploaded asynchronously. We cannot remove the file here. It's better to move the ```QFile::remove``` call somewhere to the ```LoginManager::upload```... or not. Anyway, I suggest removing the ```QFile::remove``` call from this PR and create new on with proper solution.